src/ucp/api/ucp_compat.h

```diff
@@ -33,4 +35,56 @@ void ucp_request_release(void *request);
 void ucp_ep_destroy(ucp_ep_h ep);
 
 
+/**
+ * @ingroup UCP_ENDPOINT
+ * @deprecated Replaced by @ref ucp_ep_close_nb.
+ */
+ucs_status_ptr_t ucp_disconnect_nb(ucp_ep_h ep);
```

Comment:
why need to add functions to compat.h? should be just adding C wrapper..

Reply:
really strange... when I cherry-pick this commit git displayed that there was indentation change only, no new lines.

I rebased commit again